frontend/dispatcher.lua

```diff
@@ -274,6 +276,8 @@ local dispatcher_menu_order = {
     "toggle_flash_on_chapter_boundaries",
     "set_no_flash_on_second_chapter_page",
     "toggle_no_flash_on_second_chapter_page",
+    "set_flash_on_images",
+    "toggle_flash_on_images",
```

Comment:
Here for the actions `...flash_on_images` but also for the setting name `refresh_on_images`: may be have them more verbose and explicite and long, like the others: 
```
refresh_on_chapter_boundaries      / ...flash_on_chapter_boundaries
no_refresh_on_second_chapter_page  / ...no_flash_on_second_chapter_page
refresh_on_page_with_images        / ...flash_on_page_with_images
```
so when browsing around settings and gesture files, we know they are related to pages of books, and not just a refresh on icons or images collections books like CBZ.

Reply:
Done!